modules/test/playwright/tests/frontend-data-set-views-web/dataSetsSample.spec.ts

```diff
@@ -0,0 +1,64 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {apiHelpersTest} from '../../fixtures/apiHelpersTest';
+import {featureFlagsTest} from '../../fixtures/featureFlagsTest';
+import {loginTest} from '../../fixtures/loginTest';
+import getRandomString from '../../utils/getRandomString';
+import {pageEditorPagesTest} from '../layout-content-page-editor-web/fixtures/pageEditorPagesTest';
+
+export const test = mergeTests(
+	apiHelpersTest,
+	featureFlagsTest({
+		'LPS-164563': true,
+	}),
+	loginTest(),
+	pageEditorPagesTest
+);
+
+test('Add the frontend data set sample widget', async ({apiHelpers, page}) => {
+	let site: Site;
+
+	await test.step('Create a content site and the frontend data set sample widget', async () => {
+		site = await apiHelpers.headlessSite.createSite('Test Site');
```

Comment:
We should use `isolatedSiteTest` instead of creating site. See https://github.com/liferay-frontend/liferay-portal/pull/4045